datafusion/src/physical_plan/sorts/sort_preserving_merge.rs

```diff
@@ -257,7 +257,7 @@ pub(crate) struct SortPreservingMergeStream {
     /// The sorted input streams to merge together
     streams: MergingStreams,
 
-    /// Drop helper for tasks feeding the [`receivers`](Self::receivers)
+    /// Drop helper for tasks feeding the input [`streams`](Self::streams)
```

Comment:
yeah this code has been significantly changed since the original docstring was introduced